index.js

```diff
@@ -138,6 +237,20 @@ PSPDFKitView.propTypes = {
      * Controls wheter or not the default action for tapped annotations is processed. Defaults to processing the action (false).
      */
     disableDefaultActionForTappedAnnotations: PropTypes.bool,
+    /**
+     * Controls whether or not the document will be automatically saved. Defaults to automatically saving (false).
+     * 
+     * @platform android
```

Comment:
The Android flag can be removed as the merged #99 PR adds the iOS implementation.